static/app/constants/releases.tsx

```diff
@@ -0,0 +1,11 @@
+export enum ReleasesSortOption {
```

Comment:
this enum was just moved from https://github.com/getsentry/sentry/blob/5078825918dfc42a0fd3cccbaa3643fdbc776d15/static/app/views/releases/list/releasesSortOptions.tsx#L8-L18